pkg/pillar/cmd/logmanager/logmanager.go

```diff
@@ -682,6 +682,7 @@ func sendProtoStrForLogs(reportLogs *logs.LogBundle, image string,
 	reportLogs.Timestamp = ptypes.TimestampNow()
 	reportLogs.DevID = *proto.String(devUUID.String())
 	reportLogs.Image = image
+	reportLogs.EveVersion = readEveVersion("/etc/eve-release")
```

Comment:
Don't you want to do this when logmanager starts since the content of the file never changes on a running system? Would be more efficient.

Reply:
Right, made eveVersion a global variable that is initialized in the beginning.